package/MDAnalysis/analysis/density.py

```diff
@@ -564,8 +564,10 @@ def density_from_Universe(universe, delta=1.0, atomselection='name OH2',
                           gridcenter=None, xdim=None, ydim=None, zdim=None):
     """Create a density grid from a :class:`MDAnalysis.Universe` object.
 
-    The trajectory is read, frame by frame, and the atoms selected with `atomselection` are
-    histogrammed on a grid with spacing `delta`.
+    The trajectory is read, frame by frame, and the atoms selected with
+    `atomselection` are histogrammed on a grid with spacing `delta`.
+    A physical density of units [Angstroem^{-3}] is returned (see
```

Comment:
Typo: Angstroem

Reply:
To make things easier in future efforts, I have now standardised the entire of density.py to Angstrom (matches what is declared in units.py).